index.html

```diff
@@ -0,0 +1,216 @@
+                </button>
+
+            </li>
+            <li class="item">
+                <img src="/image/Snapshoot Portfolio4.png" alt="">
+                <h2 class="tail">Multi-Post <br> Stories</h2>
+                <div class="type">
+                    <p class="t1">CANOPY</p>
+                    <img src="/images/circle.png" />
+                    <p class="t2">Back End Dev</p>
+                    <img src="/images/circle.png" />
+                    <p class="t3">2015</p>
+                </div>
+                <p class="text">A daily selection of privately personalized reads: no accounts or sign-ups required.</p>
+                <ul class="tech">
+                    <li class="lang lang1">html</li>
+                    <li class="lang lang2">css</li>
+                    <li class="lang lang3">javascript</li>
+                </ul>
+                <button class="show details">See project
+                </button>
+            </li>
+        </ul>
+    </section>
+    <section class="headline about">
+        <h2>About </br> Myself</h2>
+        <p class="details">
+            I'm a software developer! I can help you build a product, feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <h4>LET'S CONNECT</h4>
+        <div class="social">
+            <ul>
+                <li><a href="twitter.com/khanya_mvoko"><i class="fa-brands fa-square-twitter"></i></a></li>
+                <li><a href="linkedin.com/khanya-mvoko"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="github.com/Hotdog39duty"><i class="fa-brands fa-square-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <button class="seee">
+            Get my resume
+        </button>
+        <div class="fram">
+            <h4>Languages</h4>
+            <span class="material-symbols-rounded">
+                expand more
+            </span>
+        </div>
+        <div class="listframe">
+            <img src="/images/js.png" alt="js">
+            <h4>JavaScript</h4>
+        </div>
+        <div class="listframe">
+            <img src="/images/html.png" alt="js">
+            <h4>Html</h4>
+        </div>
+        <div class="listframe">
+            <img src="/images/css.png" alt="js">
+        </div>
+        <div class="'fram">
+            <h4>Frameworks</h4>
+            <span class="material-symbols-rounded">
+                chevorn_right
+            </span>
+        </div>
+        <img src="/images/Separator Bottom.png" alt="">
+        <div class="fram">
+            <h4>Skills</h4>
+            <span class="material-symbols-rounded">
+                chevorn_right
+            </span>
+        </div>
+        <img src="/images/separator Bottom.png" alt="">
```

Comment:
[OPTIONAL] It looks like most of your images are not appearing as shown below, kindly check and make sure that everything works perfectly in order to make your website more proffessional. 


![Screenshot from 2023-02-22 14-41-43](https://user-images.githubusercontent.com/65233860/220656366-866ad76c-af8e-4a09-a7a7-8ed99e5ebf81.png)